app/controllers/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+class UsersController < ApplicationController
```

Comment:
You're not doing anything with this controller nor is it your responsibility.......
rm -rf